ports/nrf/mpconfigdevice_nrf51822.h

```diff
@@ -59,3 +59,11 @@
 #define MICROPY_PY_UBINASCII               (1)
 #endif
 #endif
+
+#ifndef MICROPY_ENABLE_SOURCE_LINE
```

Comment:
It's good to keep these config options in the same order as what appears in `py/mpconfig.h`, in which case this one should go before `MICROPY_PY_ARRAY_SLICE_ASSIGN` (and after `MICROPY_VFS`).  Same goes for all additions in this PR.

Reply:
Sure, good to know. Done